multisig-transactions-monitor/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "forta-compound-finance-multisig-transaction-agent",
+  "name": "forta-compound-finance-multisig-transaction-bot",
   "version": "0.0.1",
```

Comment:
While a trivial change (and technically a new deployment based on the Bot ID), this should be incremented to version `0.0.2`.